pom.xml

```diff
@@ -58,7 +58,7 @@
     <version.com.github.javaparser>3.13.10</version.com.github.javaparser>
     <version.com.fasterxml.jackson>2.9.10</version.com.fasterxml.jackson>
     <version.kubernetes-client>4.4.2</version.kubernetes-client>
-    <version.okhttp>3.12.0</version.okhttp>
+    <version.okhttp>3.12.6</version.okhttp>
```

Comment:
@cristianonicolai we might need too revert back this one or upgrade the `kubernetes-client` version also:
https://github.com/fabric8io/kubernetes-client/blob/v4.4.2/pom.xml#L85

But this `kubernetes-client` must follow Quarkus one.

Reply:
@ricardozanini done :)